core/include/drivers/stm32_rif.h

```diff
@@ -0,0 +1,212 @@
+/* SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause) */
```

Comment:
I don't think these header files require dual licensing, it is specific to OP-TEE OS.
(contrary to DT bindings header files)

Reply:
Right, I'll rework these and remove the useless header file